dataModel/src/main/scala/za/co/absa/enceladus/model/Dataset.scala

```diff
@@ -27,26 +27,26 @@ case class Dataset(
   name:    String,
   version: Int = 1,
   description: Option[String] = None,
-  
+
   hdfsPath:        String,
   hdfsPublishPath: String,
 
   schemaName:    String,
   schemaVersion: Int,
 
   dateCreated: ZonedDateTime = ZonedDateTime.now(),
-  userCreated: String        = null,
+  userCreated: String        = null, // scalastyle:ignore null
```

Comment:
Maybe here we will want to create a new issue for removing `null`s from the models and changing them for `""`. If my knowledge is correct then it should still work fine.

Reply:
`_` would almost certainly break things. 
`""` would also very likely break things.
It should eventually be refactored to a case class which holds a date and username. But that's the least of this model's problems.